Cargo.toml

```diff
@@ -16,8 +16,9 @@ clap = { version = "4.0", features = ["derive"] }
 config = "0.13"
 git2 = "0.19"
 percent-encoding = "2.3.1"
-
+# Add this dependency for opening URLs in the default browser
+webbrowser = "0.8"
```

Comment:
Was there any reason for the package version to be set to 0.8 ?

If not, please update it to the latest version

Reply:
updated it to the latest version now